mariadb/templates/configmaps/seed.sh.yaml.j2

```diff
@@ -0,0 +1,120 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  seed.sh: |+
+    #!/bin/sh
+
+    set -ex
+
+    SLEEP_TIMEOUT=5
+    # we need to count seed instance here.
+    MINIMUM_CLUSTER_SIZE=4
```

Comment:
Let me rephrase: what happens if an user wants to simply try Stackanetes out with 2 nodes (1 control, 1 compute, `deployment.replicas=1`)? Can we do it?